CONTRIBUTING.md

```diff
@@ -72,3 +75,16 @@ Once your PR is approved and all CI checks have passed, do the following to merg
 ![Screenshot 4](https://i.imgur.com/Lne6GbI.png)
 * Set the merge commit message as the PR title. Additionally you can leave a description, that summarizes the whole PR, or leave it empty. But please do not send commit list with the description, because it takes too much space on git.
 ![Screenshot 5](https://i.imgur.com/ZuiGruB.png)
+
+# Opening an issue
+If it is related to a specific project, mark the title as such. For example:
+
+`Noloan: New button in PrettyButtonsActivity doesn't work`
```

Comment:
Sounds like a name for me. Edward Nolan. I guess it's actually one of projects. I just wonder it may confuse someone.

Reply:
Done